src/vos/tests/evt_ctl.c

```diff
@@ -2435,17 +2344,34 @@ main(int argc, char **argv)
 	ts_uma = utest_utx2uma(ts_utx);
 
 	if ((argc - optind) == 1) {
-		rc = dts_cmd_parser(ts_ops, "$ > ", ts_cmd_run);
+		/*rc = dts_cmd_parser(ts_ops, "$ > ", ts_cmd_run);*/
+		print_message("Invalid format.\n");
+		rc = 1;
 		goto out;
 	}
 
-	while ((rc = getopt_long(argc, argv, "C:a:m:e:f:g:d:b:Docl::ts:",
-				 ts_ops, NULL)) != -1) {
-		rc = ts_cmd_run(rc, optarg);
-		if (rc != 0)
-			goto out;
+	/* First, execute Internal tests in the command */
+	int j;
+	for(j=0; j<argc;j++){
+		if(strcmp(argv[j],"-t") == 0 ) {
+			rc = run_internal_tests();
+			if (rc != 0)
+				D_PRINT("Internal tests failed with rc="DF_RC"\n", DP_RC(rc));
+		}
+	}
+	
+	/* Execute the sequence of tests */
+	int start_idx;
```

Comment:
For better or worse, we always put variables at the top of the block.   And generally, we align variable names with tabs (plus a space to align a pointer type with a non pointer type)....

int[tab]*x;
int[tab] y; /* note the space where the * would be. */

This isn't consistent throughout daos but it is pretty consistent in VOS

Reply:
Okay, I'll send another PR.
Thanks Jeff!